scripts/install_deps.sh

```diff
@@ -80,3 +80,14 @@ fi
 if [[ "${MACHINE}" == "Mac"  ]]; then
     brew install boost libomp llvm ninja tbb openblas
 fi
+
+if [[ "${MACHINE}" == "MinGw"  ]]; then
+    pacman -Su --noconfirm --needed \
+    git make tar dos2unix zip unzip patch \
+    mingw-w64-x86_64-toolchain \
+    mingw-w64-x86_64-make \
+    mingw-w64-x86_64-cmake \
+    mingw-w64-x86_64-boost \
```

Comment:
knowhere has no dependency on boost and tbb, I have remove boost and tbb for ubuntu, centos and mac

Reply:
I'll remove it in the next rebased version.